src/features/moviesList/styled.js

```diff
@@ -4,6 +4,7 @@ import { Link } from 'react-router-dom';
 export const Wrapper = styled.div`  
   display: grid;
   grid-template-columns: repeat(auto-fill, minmax(324px, 1fr));
+  grid-template-rows: auto;
```

Comment:
I don't think it changes anything

Reply:
hmm... I'll remove it now, but we'll see how it works in production